web/components/view/ShowHideTextView/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import arrow from '../../../assets/arrow.svg';
+
+class ShowHideTextView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {isToggleOn: true};
+    this.handleClick = this.handleClick.bind(this);
```

Comment:
are you sure we need this?

Reply:
https://github.com/nwhacks/nwhacks2019/pull/14

Reply:
oh so https://github.com/nwhacks/nwhacks2019/pull/96/files#diff-5adcdc976f474b7fc3770390b8a7fb4aR12 should be `handleClick = () => { ... }`

Reply:
yep!

Reply:
Whoops I completely forgot about that, will fix